lib/page_steps.js

```diff
@@ -47,18 +47,20 @@ step("Check text <text> exists", async text => {
 });
 
 step("Click <eleType> with text <text>", async (eleType, text) => {
-  await taiko.click(await selectors.getElementBySelector(eleType, text));
+  const ele = await selectors.getElementBySelector(eleType, text);
+  await taiko.evaluate(ele, (item) => item.scrollIntoView());
```

Comment:
scroll into view has to be achieved with separate step `* Scroll to "element"`

Reply:
We noticed this issue only after upgrading to the latest taiko version. It was working fine with the prev verison. If we use *Scroll to "element", there are so many places to be modified. So we had a chat with Murali and he said it is a good approach to tackle it,  particularly relevant when the element is not in view (due to size within view port, and needed scroll)